Vungle/src/main/java/com/applovin/mediation/adapters/VungleMediationAdapter.java

```diff
@@ -403,19 +403,44 @@ private void updateUserPrivacySettings(final MaxAdapterParameters parameters)
         }
     }
 
-    private static BannerAdSize vungleAdSize(final MaxAdFormat adFormat)
+    private VungleAdSize vungleAdSize(
+        final MaxAdFormat adFormat,
+        final MaxAdapterParameters parameters)
     {
-        if ( adFormat == MaxAdFormat.BANNER )
+        final Map<String, Object> localExtraParameters = parameters.getLocalExtraParameters();
+        Object isAdaptiveObj = localExtraParameters.get( "adaptive_banner" );
+        boolean isAdaptiveBanner = false;
+        if ( isAdaptiveObj instanceof String ) {
```

Comment:
Do you still support adaptive here?

Reply:
No, just to check adaptive_banner should be false for MAX custom size.
I have no idea if we need to check this field or is it ok only check the adaptive_banner_width and adaptive_banner_height? This depend on MAX integration guide doc.